core/modules/ckeditor5/ckeditor5.module

```diff
@@ -607,6 +607,7 @@ function ckeditor5_ckeditor5_plugins() {
           'image.ImageInsertUI',
           'image.ImageUpload',
           'image.ImageResize',
+          'image.ImageCaption',
```

Comment:
I think we can switch to the specific subplugin rather than the whole ImageCaption. I think this may be better to prevent image caption UI from showing up when captions are not supported.

```suggestion
          'image.ImageCaptionUtils',
```